Content.Server/_Harmony/GameTicking/Rules/ConspiratorRuleSystem.cs

```diff
@@ -0,0 +1,98 @@
+using Content.Server._Harmony.GameTicking.Rules.Components;
+using Content.Server.Antag;
+using Content.Server.GameTicking;
+using Content.Server.GameTicking.Rules;
+using Content.Server.Roles;
+using Content.Shared._Harmony.Conspirators.Components;
+using Content.Shared._Harmony.Roles.Components;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Mind;
+using Content.Shared.Objectives;
+using Content.Shared.Objectives.Systems;
+using Content.Shared.Random;
+using Content.Shared.Random.Helpers;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Utility;
+
+namespace Content.Server._Harmony.GameTicking.Rules;
+
+public sealed class ConspiratorRuleSystem : GameRuleSystem<ConspiratorRuleComponent>
+{
+
+    [Dependency] private readonly AntagSelectionSystem _antag = default!;
+    [Dependency] private readonly SharedMindSystem _mind = default!;
+    [Dependency] private readonly SharedObjectivesSystem _objectives = default!;
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<ConspiratorRoleComponent, GetBriefingEvent>(OnGetBriefing);
+        SubscribeLocalEvent<ConspiratorRuleComponent, AfterAntagEntitySelectedEvent>(OnAntagSelected);
+    }
+
+    protected override void AppendRoundEndText(EntityUid uid,
+        ConspiratorRuleComponent component,
+        GameRuleComponent gameRule,
+        ref RoundEndTextAppendEvent args)
+    {
+        base.AppendRoundEndText(uid, component, gameRule, ref args);
+
+        var sessionData = _antag.GetAntagIdentifiers(uid);
+        args.AddLine(Loc.GetString("conspirator-count", ("count", sessionData.Count)));
+        foreach (var (mind, data, name) in sessionData)
```

Comment:
The mind variable is unused, please set the name as '_' in order to suppress the warning.
```suggestion
        foreach (var (_, data, name) in sessionData)
```